source/doc-pages/guides/basics/relations.md

```diff
@@ -0,0 +1,172 @@
+# Relations
+
+In ROM relations are the central part of the system. They provide access to data
+through internal interfaces provided by adapters and allow you to specify your
+own interface for accessing application-specific data structures.
+
+The fundamental design principle in ROM is that all components work with relations
+rather than individual objects from relations. This means that a relation can only
+return other relations (also known as "views" or "virtual relations"). Then both
+commands and mappers receive a relation, do their work and return a relation back.
+
+## Defining a relation
+
+A relation can be defined as a class that inherits from an adapter-specific class
+which is identified by adapter's name, in case of our example it's `:memory`:
+
+``` ruby
+class Users < ROM::Relation[:memory]
+end
+```
+
+Alternatively you can use the routing-style DSL:
+
+```ruby
+ROM.relation(:users) do
```

Comment:
Might want to show how you specify the memory adapter here, so it's a more clear parallel to the class definition above.